indico/core/signals/event/designer.py

```diff
@@ -22,3 +22,8 @@
 Called when printing a badge template. Registration form is passed in the 'regform' kwarg.
 kwarg.
 """)
+
+generate_ticket_qr_code = _signals.signal('generate-ticket-qr-code', """
+Called when generating a ticket QR code. QR object and QR data are p
```

Comment:
`p`? Also, mentioning kwargs without mentioning their names is a bit pointless ;)